packages/toolkit/.changes/next-release/Feature-c5435979-4c37-4dce-8ebb-dd14b77d3b87.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Allow customers to local invoke with the runtime selected in the local invoke webview"
```

Customers are the primary audience for changelogs.  Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).
```suggestion
	"description": "AppBuilder: local invoke with the runtime selected in the local invoke webview"
```